webofneeds/won-matcher-sparql/src/main/java/won/matcher/sparql/actor/SparqlMatcherActor.java

```diff
@@ -149,6 +149,7 @@ public void onReceive(final Object o) throws Exception {
             log.info(String.format("Caught exception when processing %s event %s. More info on loglevel 'debug'",
                     eventTypeForLogging, uriForLogging.orElse("[no uri available]")));
             log.debug("caught exception", e);
+            e.printStackTrace();
```

Comment:
this needs to be removed before merge

Reply:
Wrapped it in an if-clause so it actually only prints the trace in the debug log, not the regular log.

Reply:
if clause would be executed if the debug log level is set, but not that it would print it in our debuglog, printStackTrace circumvents the logger entirely